modules/nw-ovn-k-day-2-masq-subnet.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configure-ovn-kubernetes-subnets.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ovn-k-day-2-masq-subnet_{context}"]
+= Configuring the OVN-Kubernetes masquerade subnet as a day two operation
+
+You can change the masquerade subnet used by OVN-Kubernetes as a day two operation in order to avoid conflicts with any existing subnets that are already in use in your environment.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Change your cluster's masquerade subnet:
+
+** For dualstack clusters using IPv6, run the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"ipv4":{"internalMasqueradeSubnet": "<masquerade_subnet>"},"ipv6":{"internalMasqueradeSubnet": "<masquerade_subnet>"}}}}}}'
+----
++
+Where:
+
+`internalMasqueradeSubnet`:: Specifies an IP address to be used as the masquerade subnet. This range cannot overlap with any other subnets used by {product-title} or on the host itself. The default values for IPv4 is `100.254.0.0/17` and for IPv6 the default value is
```

Comment:
The default value for ipv4 and ipv6 is not correct, see here https://github.com/openshift/ovn-kubernetes/blob/master/go-controller/pkg/config/config.go#L145-L146